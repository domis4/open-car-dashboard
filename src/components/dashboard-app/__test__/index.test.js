import React from 'react'
import { shallow } from 'enzyme'
import Dashboard from '../index'

describe('dashboard-app component index.js', () => {
  it('expects component to match snapshot', () => {
    const wrapper = shallow(<Dashboard />).dive()

    expect(wrapper).toMatchSnapshot()
  })
})
