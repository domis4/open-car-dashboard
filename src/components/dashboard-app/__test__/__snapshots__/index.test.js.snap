// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dashboard-app component index.js expects component to match snapshot 1`] = `
<WithTheme(Zoom)
  in={true}
>
  <WithStyles(Card)
    onClick={[Function]}
  >
    <WithStyles(CardActionArea)>
      <WithStyles(CardMedia)
        className="View-media-01"
        component="img"
        image={
          Object {
            "process": [Function],
          }
        }
      />
      <WithStyles(CardContent)>
        <WithStyles(Typography)
          className="View-text-02"
          component="h2"
          gutterBottom={true}
          variant="headline"
        >
          App Title
        </WithStyles(Typography)>
      </WithStyles(CardContent)>
    </WithStyles(CardActionArea)>
  </WithStyles(Card)>
</WithTheme(Zoom)>
`;
